reference/content-types/single_category_selection.rst

```diff
@@ -0,0 +1,33 @@
+Single Category selection
+=========================
+
+Description
+-----------
+
+Let you assign one category. Categories can be managed in the settings section of Sulu.
+The selection will be saved as a single id.
+
+Parameters
+----------
+
+.. list-table::
+    :header-rows: 1
+
+    * - Parameter
```

Comment:
this should also include the `item_disabled_condition` and `item_disabled_condition`